.github/multi-labeler.yml

```diff
@@ -20,26 +22,38 @@ labels:
   - label: 'chore'
     matcher:
       title: '^chore(\(|:)'
-  # "change",
   - label: 'docs'
     matcher:
       title: '^docs(\(|:)'
-  # "style",
   - label: 'refactor'
     matcher:
       title: '^refactor(\(|:)'
-  # "test",
   - label: 'auto'
     matcher:
       title: '^auto(\(|:)'
 
-  # Below are the scopes that we look for in the PR title
+  # additional meta flags
+
+  # note, this does not pick up chained PRs automatically
+  - label: 'stable'
+    matcher:
+      baseBranch: '^stable-.+'
+
+  - label: 'cherry-pick'
+    matcher:
+      title: '(🍒|:cherries:)'
+
+  # Scopes that we look for in the PR title
+
   - label: 'android/'
     matcher:
       title: '\(.*android.*\):'
   - label: 'common/'
     matcher:
       title: '\(.*common.*\):'
+  - label: 'core/'
```

Comment:
`core/` does not exist on `stable-15.0`, does it?

Reply:
The label exists, and the `(core)` title can be used. Just the folder is common/core/desktop/. So I think this labeler rule should be fine?

Reply:
Ah, right, it's against the title, not the path.  That's what got me.  👍